### PR TITLE
Twitter Time line fixed

### PR DESCRIPTION
**Pull request by DigiGoon**
No need of using widget ID

### CODE REVIEW

File: pelican-bootstrap3/templates/includes/twitter_timeline.html [hunk 1]
```diff
@@ -2,9 +2,9 @@
...
     <div id="twitter_timeline">
-        <a class="twitter-timeline" data-chrome="noheader" href="https://twitter.com/{{ TWITTER_USERNAME }}" data-widget-id="{{TWITTER_WIDGET_ID}}">Tweets by {{TWITTER_USERNAME}}</a>
+        <a class="twitter-timeline" data-chrome="noheader" href="https://twitter.com/{{ TWITTER_USERNAME }}">Tweets by {{ TWITTER_USERNAME }}</a> <script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
     </div>
 
 <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+"://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script>
 
-{% endif %}
\ No newline at end of file
+{% endif %}
```
Comment (TimothyBramlett) [inline pelican-bootstrap3/templates/includes/twitter_timeline.html]: Hey, thanks!  This helped me modify my own theme to account for what I assume must be recent Twitter changes with regard to Widgets, etc.

One thing I did have to change though is I changed line 1 from ` {% if TWITTER_WIDGET_ID %}` to `{% if TWITTER_TIMELINE %}`

**Comment by justinmayer** [comment]
@DigiGoon: Would you please resolve conflicts / rebase on current master?

**Comment by almet** [comment]
Hi, sorry for the delay on this, could you consider rebasing your changes on top of the latest master? Thanks!

**Comment by DigiGoon** [comment]
@almet F.. sorry, I completely forgot about this, and didn't pay attention to the notification, I hope this helped.

**Comment by almet** [comment]
Thanks, but it seems too old now, heh :-)

**Pull request closed without merging**